subsys/net/lib/download_client/Kconfig

```diff
@@ -11,13 +11,15 @@ if NRF_DOWNLOAD_CLIENT
 
 config NRF_DOWNLOAD_MAX_FRAGMENT_SIZE
 	int "Fragment size"
-	default 2048
+	range 512 2048 if BSD_LIBRARY && NRF_DOWNLOAD_CLIENT_TLS
+	default 2048 if BSD_LIBRARY && NRF_DOWNLOAD_CLIENT_TLS
+	default 4096
```

Comment:
Also, this is an increase, compared to before. We need to have an explanation for that.

Reply:
Would you like a separate commit for that or can I add some text in `help` to explain why it is better to have a larger value when not using TLS?

Reply:
I think it's fine to do it in the same commit that the change was introduced.

Reply:
Added a sentence in `help`.

Reply:
thanks